Cargo.toml

```diff
@@ -50,9 +50,6 @@ members = [
 resolver = "2"
 
 [patch.crates-io]
```

Comment:
This section doesn't seem to be required anymore.
```suggestion
```

Reply:
fixed